media-libs/libvpx/libvpx1.11-1.11.0.recipe

```diff
@@ -27,12 +23,12 @@ if [ "$targetArchitecture" = x86_gcc2 ]; then
 	commandBinDir=$prefix/bin
 fi
 
-libVersion="6.4.0"
+libVersion="7.0.0"
 libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 portVersionCompat="$portVersion compat >= ${portVersion%.*}"
 
 PROVIDES="
-	libvpx6$secondaryArchSuffix = $portVersion
+	libvpx1.11$secondaryArchSuffix = $portVersion compat >= 1.11
```

Comment:
The compat shouldn't be needed, SONAME changes, so it can't be both "7" or "8"
```diff
-	libvpx1.11$secondaryArchSuffix = $portVersion compat >= 1.11
+	libvpx1.11$secondaryArchSuffix = $portVersion
```
Also line 28 can be removed, isn't used in the version.